### PR TITLE
minor readme update

### LINKED ISSUE #91: It should be sass instead of scss
## Describe the bug

In readme sass instead of scss should be written
Screenshot from 2021-10-03 11-03-41

### PR DESCRIPTION
**Pull request by parzival272000**
**Issue Number**

fixes #91 

**Describe the changes you've made**
Updated the readme

**Checklist**

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.

**Provide a Deployed link of route/page that needs to review**

Preview: Deploy preview link here with the appropriate route

<!-- Please add deployed link to here text >

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +40,8 @@ The page will reload if you make edits. You will also see any lint errors in the
 Before submitting a PR/MR make sure your code is compliant with our JS rules by running: `npm run test:js`
 You can format it automatically by running: `npm run format`.
 
-To make sure your SCSS is compliant run: `npm run test:scss`
+We use the SASS linter because it is also compatible with SCSS syntax.To make sure your SCSS is compliant
```
Comment (cyntss) [inline README.md]: can you add a space after the first full stop?

`syntax. To`

File: package.json [hunk 2]
```diff
@@ -53,6 +53,7 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
+    "test:scss": "sass-lint -v -q",
     "test:sass": "sass-lint -v -q",
```
Comment (cyntss) [inline package.json:57]: can you remove the old script `"test:sass": "sass-lint -v -q",` ?

**Review by cyntss — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (parzival272000) [reply package.json:57]: yeah but without that it was giving error in the checks online tests and also in the local when one enters npm run test:sass

File: README.md (hunk 1 above)
Comment (parzival272000) [reply README.md]: Added

File: package.json (hunk 2 above)
Comment (cyntss) [reply package.json:57]: got it.. we need to update the tests too. Let's merge this and we'll update the rest in another PR

Comment (parzival272000) [reply package.json:57]: Ok, looking forward to it.

**Pull request merged**